plugin/src/main/java/com/denizenscript/denizen/events/entity/EntityEntersBlockScriptEvent.java

```diff
@@ -0,0 +1,87 @@
+package com.denizenscript.denizen.events.entity;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizen.objects.MaterialTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.EntityEnterBlockEvent;
+
+public class EntityEntersBlockScriptEvent extends BukkitScriptEvent implements Listener {

```

Comment:
class name should ideally match the event name, since the event was renamed to `goes into` the class should be too